### ISSUE TITLE
Nether build height reduced in 1.13 update.

### ISSUE THREAD

**Issue by Andro609** [rlf/uSkyBlock#1172]
Before 1.13 version of USB could build up to _and including_ **y=120**
After 1.13 version of USB can only build up to  **y=119**

Breaks builds/farms/etc...

any advise on how to fix would be apreciated. I did search through the config files but could not see and nether height options.

/usb version
Sorry don't have this, not my server. Trying to find fix to problem.

**Comment by Muspah** [comment]
You're absolutely right, the max Y for the WorldGuard nether regions changed from 120 to 119 here:

https://github.com/rlf/uSkyBlock/commit/26095cd10809c5fab0261d209c1ee1b1f63ff47c#diff-81616fe4dc052409e8325ebe439c21abL126

I'm not sure if there's a reason behind this, so bare with me for the moment, I'll look it up and do some testing.

**Comment by Muspah** [comment]
As far as I can tell, there's no specific reason to lower the limit from 120 to 119. I've changed the max Y for the nether regions back to y120. It should be included in the next update published on SpigotMC.

If the admin of the server that you're playing on, or someone else reading this, wants to test it out, there is a pre-compiled build available for download here: https://oss.pellegrom.pt/snapshots/uSkyBlock/uSkyBlock-20190510-1-181f70fa.jar

**Comment by Andro609** [comment]
In case there is a reason for changing those figures in new versions, (island gen ?), wouldn't it be better to move them to one of the config files ?

Similarly the change from 3 to 6 on the minimum build. I know people glassing over the bedrock, who now have islands stuck as half glass, half lava.
